src/sentry/snuba/metrics/utils.py

```diff
@@ -101,6 +99,24 @@
 }
 
 
+def generate_operation_regex():
+    """
+    Generates a regex of all supported operations defined in OP_TO_SNUBA_FUNCTION
+    """
+    op_lst = []
+    for item in OP_TO_SNUBA_FUNCTION.values():
+        op_lst += list(item.keys())
+
+    op_regex = r"("
+    for op in op_lst:
+        op_regex += f"{op}|"
+    op_regex = op_regex[0:-1] + ")"
```

Comment:
```suggestion
    op_regex = op_regex[:-1] + ")"
```
`0` is default. so you can avoid it.